.github/workflows/ci.yaml

```diff
@@ -108,14 +108,23 @@ jobs:
             echo Skipping unstable tests
             echo "mark_expression=not unstable" >> $GITHUB_OUTPUT
           fi
-      - name: Run integration tests
-        # set a predictable model name so it can be consumed by charm-logdump-action
-        run: sg snap_microk8s -c "tox run -e ${{ matrix.tox-environments }} -- --model testing -m '${{ steps.select-tests.outputs.mark_expression }}'"
-        env:
-          CI_PACKED_CHARMS: ${{ needs.build.outputs.charms }}
-      - name: Dump logs
-        uses: canonical/charm-logdump-action@main
-        if: failure()
+
+      - name: Setup tmate session
+        uses: mxschmitt/action-tmate@v3
         with:
-          app: mongodb-k8s
-          model: testing
+          limit-access-to-actor: true
+        if: ${{ failure() }}
+
+      # - name: Run integration tests
+      #   # set a predictable model name so it can be consumed by charm-logdump-action
+      #   run: sg snap_microk8s -c "tox run -e ${{ matrix.tox-environments }} -- --model testing -m '${{ steps.select-tests.outputs.mark_expression }}'"
+      #   env:
+      #     CI_PACKED_CHARMS: ${{ needs.build.outputs.charms }}
+
+
+      # - name: Dump logs
+      #   uses: canonical/charm-logdump-action@main
+      #   if: failure()
+      #   with:
+      #     app: mongodb-k8s
```

Comment:
wait I am a noob, I think this fixes + disables them.

Shouldn't it only fix them? Or did I miss something during the call today

Reply:
This is done on purpose for debug tests on CI (which I am doing at the moment) :)